### PR TITLE
Feat/next

### PR DESCRIPTION
**Pull request by sabisin**
Initial dev using next.js.

Switch pages settings to action & merging to main will deploy to pages.

Legal links are not included (PP & CP) as we don't have any tracking. We can add'em if you plan to add analytics, even then we can avoid legal if we do it in anonymous mode.

Preview - https://sabisin.github.io/spatial-energy/

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
```
Comment (grudelsud) [inline pages/_app.tsx:1]: can we please add the GA tag:

```<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-WSLCVBQ8SE"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-WSLCVBQ8SE');
</script>```

File: u9/components/Menu/Menu.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import { useContext } from "react";
+import { BaseContext, BaseContextType } from "u9/contexts/base";
+import { colors } from "u9/utils/styles/theme";
+import Logo from "../Logo/Logo";
+import * as Styled from "./Menu.styles";
+
+type MenuProps = {
+  isDark?: boolean;
+};
+const Menu = ({ ...props }: MenuProps) => {
```
Comment (grudelsud) [inline u9/components/Menu/Menu.tsx:15]: I can't access the about and services pages when on mobile, is it possible to add the simplest possible menu to give access to such pages?

localhost_3000_(Pixel 5)

File: u9/containers/Services/Services.tsx [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+      },
+      {
+        phase: "Phase 2",
+        title: "Production of MVP",
+        description:
+          "Nullam vulputate gravida enim, et tincidunt augue condimentum at. Nullam ultricies felis nisl, vitae pellen venenatis quis. Vitae vestibulum arcu, a semper metus nullam placerat.",
+        image: "./images/services-phase-2.png",
+      },
+      {
+        phase: "Phase 3",
```
Comment (grudelsud) [inline u9/containers/Services/Services.tsx:75]: phase 3 and 4 come out as grey, both text and images. there's a "non greyed" version of such images in figma, can we please replace them, and have the text the same color of phase 1 and 2?

File: u9/containers/About/About.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+                the construction and real estate industry.
+              </div>
+            </div>
+          </Styled.TopContent>
+        </WrapperInner>
+        <WrapperInner spaceSize={20}>
+        <Styled.CaseStudies>
+          <Styled.Title>Case Studies</Styled.Title>
+          <Styled.Acc>
+            {data.map((item, index) => (
```
Comment (grudelsud) [inline u9/containers/About/About.tsx]: if it's not too much effort, can we have the same accordion as it's been implemented in the services page, please?

**Review by grudelsud — CHANGES_REQUESTED** [review]
thanks @sabisin 😄 I've added a few comments as discussed

**Comment by sabisin** [comment]
@grudelsud I have included the feedback. ✌️

**Review by grudelsud — APPROVED** [review]

**Pull request merged**